docs/spec/poa/README.md

```diff
@@ -0,0 +1,29 @@
+# POA module specification
+
+## Overview
+
+The POA module described here is meant to be used as an alternative and less complex module then the original staking located [here](../staking). In this system full nodes can become validators with out having a token at stake. The security model of a public blockchain using the POA should be based on reputation. If you would like to accept all applications to be a validator then you can turn `acceptAllValidators` to true.
```

Comment:
> should be based on reputation

`should be based on the governance curated registry that defines validator set and assigns corresponding voting power of each individual validator `


In the basic scenario voting power is equal among the entire validator set.